src/test/scala/io/iohk/ethereum/jsonrpc/JsonRpcControllerEthSpec.scala

```diff
@@ -17,22 +19,22 @@ import io.iohk.ethereum.jsonrpc.PersonalService._
 import io.iohk.ethereum.ommers.OmmersPool
 import io.iohk.ethereum.ommers.OmmersPool.Ommers
 import io.iohk.ethereum.transactions.PendingTransactionsManager
-import io.iohk.ethereum.{Fixtures, LongPatience, Timeouts}
+import io.iohk.ethereum.{Fixtures, LongPatience, Timeouts, WithActorSystemShutDown}
 import org.bouncycastle.util.encoders.Hex
 import org.json4s.JsonAST._
 import org.json4s.JsonDSL._
 import org.json4s.{DefaultFormats, Extraction, Formats}
 import org.scalatest.concurrent.{Eventually, ScalaFutures}
-import org.scalatest.flatspec.AnyFlatSpec
-import org.scalatest.matchers.should.Matchers
+import org.scalatest.flatspec.AnyFlatSpecLike
 import org.scalatestplus.scalacheck.ScalaCheckPropertyChecks
 
 import scala.concurrent.Future
 
 // scalastyle:off magic.number
 class JsonRpcControllerEthSpec
-    extends AnyFlatSpec
-    with Matchers
+    extends TestKit(ActorSystem("JsonRpcControllerSpec_System"))
```

Comment:
We should use different names for each spec. `JsonRpcControllerEthSpec_System`

Reply:
Done: https://github.com/input-output-hk/mantis/pull/751/commits/1e9b0bec3cb8eb9436425a2e362ec749d826a241